### PR TITLE
Readme.md: add a more gentle introduction and a nice logo

### PR DESCRIPTION
**Pull request by jonenst**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Improve the README.md

* **What is the current behavior?** (You can also link to an open issue here)
A user is kind of lost compiling c++ and downloading dependencies when most of the time a simple java build is enough

* **What is the new behavior (if this is a feature change)?**
The simple java build is described.

* **Other information**:
Feel free to continue building on this work and improve the wording !

### CODE REVIEW

**Comment by jonenst** [comment]
@MioRtia is good with words :) what do you think ?

File: README.md [hunk 1]
```diff
@@ -8,12 +8,37 @@
...
+## Simple Java Build
+
   * JDK *(1.8 or greater)*
   * Maven *(3.3.9 or greater)*
+
+Simply run
+```
+$ mvn package
+```
+to run all the tests. Modify some existing tests or create your own new tests to experiment with the framework ! If it suits you better, import the project in an IDE and use the IDE to launch your own main classes. If you know java and maven and want to do things manually, you can also use maven directly to compute the classpath of all the project jars and run anything you want with it.
```
Comment (mathbagu) [inline README.md]: Small typo:

> framework!

File: README.md [hunk 2]
```diff
@@ -8,12 +8,37 @@
...
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
+
+![PowSyBl Logo](https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true)
+
+Read more at http://www.powsybl.com !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
+
+## Project Structure
```
Comment (mathbagu) [inline README.md]: Should we have a `Getting started` section to redirect people to the [itools](https://powsybl.github.io/docs/installation/itools-packager.html) and [javaFX](https://powsybl.github.io/docs/installation/javafx-packager.html) documentation pages before explaining how to build powsybl from sources?

File: README.md [hunk 3]
```diff
@@ -8,12 +8,37 @@
 [![MPL-2.0 License](https://img.shields.io/badge/license-MPL_2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
 [![Join the chat at https://gitter.im/powsybl/powsybl-core](https://badges.gitter.im/powsybl/powsybl-core.svg)](https://gitter.im/powsybl/powsybl-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
 
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
```
Comment (MioRtia) [inline README.md]: Maybe quickly mention explicitly the LF Energy and add a link to it ? (www.lfenergy.org)

File: README.md [hunk 4]
```diff
@@ -8,12 +8,37 @@
...
+
+![PowSyBl Logo](https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true)
+
+Read more at http://www.powsybl.com !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
```
Comment (MioRtia) [inline README.md]: typo: *This project and everyone participating in it **are** governed

Comment (jonenst) [reply README.md]: I just lifted the phrase from the contributor covenant. The variant with "is" wins a google fight 400-2  against the variant with "are" :)

https://www.google.com/search?q=%22This+project+and+everyone+participating+in+it+are+governed+by%22
https://www.google.com/search?q=%22This+project+and+everyone+participating+in+it+is+governed+by%22

File: README.md (hunk 1 above)
Comment (jonenst) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (jonenst) [reply README.md]: I reworded the intro and talked about downloading demos (which we have to build) and getting guidance on starting you own app with a link to doc/tutorials

File: README.md (hunk 3 above)
Comment (jonenst) [reply README.md]: Yes good idea

[the remark this answers is not in the document]
Comment (jonenst) [reply README.md]: Good idea, added

File: README.md [hunk 5]
```diff
@@ -8,12 +8,37 @@
 [![MPL-2.0 License](https://img.shields.io/badge/license-MPL_2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
 [![Join the chat at https://gitter.im/powsybl/powsybl-core](https://badges.gitter.im/powsybl/powsybl-core.svg)](https://gitter.im/powsybl/powsybl-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
 
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
+
+![PowSyBl Logo](https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true)
```
[the remark this answers is not in the document]
Comment (jonenst) [reply README.md]: Now 50% !

File: README.md [hunk 6]
```diff
@@ -8,12 +8,37 @@
...
+
+Read more at http://www.powsybl.com !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
+
+## Project Structure
+
+The powsybl-core project provides code and tools to build applications. It is not a standalone project. However, you can still easily explore the project with this repository and experiment by running some code. Alternatively, to build a standalone GUI application, please visit https://github.com/powsybl/powsybl-gse .
```
[the remark this answers is not in the document]
Comment (jonenst) [reply README.md]: Reworded the intro to address this

**Pull request merged**